css/_aui_reset.scss

```diff
@@ -48,6 +48,7 @@ canvas,
 progress,
 video {
   display: inline-block;
+  transition: object-position 0.5s ease 0s;
```

Comment:
it won't do anything since object-position won't change for those cases. As it is now, it just ensures that whenever a video with object-fit: cover has its object-position updated, a transition will be applied